guides/common/modules/proc_upgrading-smartproxy-server.adoc

```diff
@@ -139,3 +139,28 @@ include::snip_steps-needs-reboot.adoc[]
 ifdef::orcharhino,satellite[]
 . Optional: If you use custom repositories, ensure that you enable these custom repositories after the upgrade completes.
 endif::[]
+
+.Upgrading {SmartProxyServersTitle} using remote execution
```

Comment:
IIRC, the mod docs ref guide says that a procedure module should not contain multiple procedures. This module is already extremely long and is starting to look like an assembly. I recommend creating a separate module for this procedure.

Reply:
Excellent!